src/pages/billing/List.js

```diff
@@ -267,21 +287,37 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
         div({ style: Style.breadcrumb.textUnderBreadcrumb }, [selectedName])
       ])
     ]),
-    div({ role: 'main', style: { display: 'flex', flex: 1, position: 'relative' } }, [
-      div({ style: { width: 330, boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)' } }, [
-        div({ style: Style.navList.heading }, [
+    div({ role: 'main', style: { display: 'flex', flex: 1 } }, [
+      div({ style: { minWidth: billingProjectListWidth, maxWidth: billingProjectListWidth, boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)', height: `calc(100vh - ${topBarHeight}px)`, position: 'fixed', overflowY: 'scroll' } }, [
```

Comment:
Hmm, this is a tricky one. There are some assumptions in the top level wrappers that make this difficult. Not comprehensive, but a couple suggestions:
* The `position: fixed` is probably not necessary
* The righthand container also needs to be independently scrollable in this kind of layout
* Be sure to test in the BioDataCatalyst site skin, since that has a persistent footer that will interact with this.

Reply:
Good calls, all around! `position: fixed` ended up not necessary, I think the righthand container being scrollable looks and feels better (otherwise it scrolled with the page, leaving the side list behind. The biodatacatalyst skin does have the issue of adding its own scrollbar because of the height issues. I am going to work around this by making a `pageHeightWithoutFooterOrHeader` function that takes that into account.